Tenduke.Client.WinFormsSample/Tenduke.Client.WinFormsSample.csproj

```diff
@@ -23,18 +23,15 @@
     </None>
   </ItemGroup>
   <ItemGroup>
-    <PackageReference Update="chromiumembeddedframework.runtime.win-arm64" Version="126.2.7" />
+    <PackageReference Include="chromiumembeddedframework.runtime.win-arm64" Version="135.0.17" />
   </ItemGroup>
   <ItemGroup>
-    <PackageReference Update="chromiumembeddedframework.runtime.win-x64" Version="126.2.7" />
+    <PackageReference Include="chromiumembeddedframework.runtime.win-x64" Version="135.0.17" />
   </ItemGroup>
   <ItemGroup>
-    <PackageReference Update="chromiumembeddedframework.runtime.win-x86" Version="126.2.7" />
+    <PackageReference Include="chromiumembeddedframework.runtime.win-x86" Version="135.0.17" />

```

Comment:
Same comment as for WPF. Would be good to build and test without the direct reference and see if CEF can be used as a transitive dependency only.

Reply:
I have modified this in the last commit